build.bat

```diff
@@ -2,4 +2,4 @@
 if "%1"=="" SET CONFIG=Release
 if NOT "%1"=="" SET CONFIG=%1
 
-"C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe" LunaDll.sln /m /nologo /p:Configuration="%CONFIG%" /p:Platfprm="Win32"
+"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin\amd64\MSBuild.exe" LunaDll.sln /m /nologo /p:Configuration="%CONFIG%" /p:Platfprm="Win32"
```

Comment:
Probably shouldn't check in changes for local paths in this.

Reply:
Ack, my bad. Forgot to exclude that file.